### PR TITLE
438 create dashboard page

### PR DESCRIPTION
**Pull request by JackVCurtis**
Screenshot 2023-11-27 at 12 58 20 PM

### CODE REVIEW

File: views/pages/DashboardPage/DashboardPage.test.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        iconFileName: 'profileImage.png',
+        uid: '1234'
+      }
+    }
+  }
+}))
+jest.mock('../../api', () => ({
+  participants: {
+    all: () => Promise.resolve([
+            {
```
Comment (mikestone14) [inline views/pages/DashboardPage/DashboardPage.test.jsx]: I think it's worth making this a fixture along with the others in `test/fixtures.js`. That way if/when other attributes are added we only need them added in 1 place.

File: views/pages/DashboardPage/DashboardPage.test.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          </AuthContext.Provider>
+        </NotificationContext.Provider>
+      </MemoryRouter>
+    )
+
+    await waitFor(() => {
+      screen.getByText('Dashboard')
+    })
+    const dashboardTitle = screen.getByText('Dashboard')
+    expect(dashboardTitle).toBeInTheDocument()
```
Comment (mikestone14) [inline views/pages/DashboardPage/DashboardPage.test.jsx]:
```suggestion
    await waitFor(() => {
      expect(screen.getByText('Dashboard')).toBeInTheDocument()
    })
```

File: views/pages/DashboardPage/DashboardPage.test.jsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        </NotificationContext.Provider>
+      </MemoryRouter>
+    )
+
+    await waitFor(() => {
+      screen.getByText('Dashboard')
+    })
+
+    const participantRows = within(screen.getAllByRole('table')[0]).getAllByRole('row')
+    expect(participantRows.length).toBe(6)
```
Comment (mikestone14) [inline views/pages/DashboardPage/DashboardPage.test.jsx]:
```suggestion
    await waitFor(() => {
      const participantRows = within(screen.getAllByRole('table')[0]).getAllByRole('row')
      expect(participantRows.length).toBe(6)
    })
```

File: views/tables/Table/index.jsx [hunk 4]
```diff
@@ -8,6 +8,7 @@ import {
 } from '@mui/material'
 
 import TableHead from './TableHead'
+import { max } from 'lodash'
```
Comment (mikestone14) [inline views/tables/Table/index.jsx]: do we need this import?

**Review by mikestone14 — COMMENTED** [review]
a couple minor things but overall this looks great

File: views/pages/DashboardPage/DashboardPage.test.jsx (hunk 1 above)
Comment (JackVCurtis) [reply views/pages/DashboardPage/DashboardPage.test.jsx]: Totally agree, thanks!

File: views/tables/Table/index.jsx (hunk 4 above)
Comment (JackVCurtis) [reply views/tables/Table/index.jsx]: No must be an accidentally IDE import. Will fix!

File: views/pages/DashboardPage/DashboardPage.test.jsx (hunk 3 above)
Comment (JackVCurtis) [reply views/pages/DashboardPage/DashboardPage.test.jsx]: 3433ed5

File: views/pages/DashboardPage/DashboardPage.test.jsx (hunk 2 above)
Comment (JackVCurtis) [reply views/pages/DashboardPage/DashboardPage.test.jsx]: 3433ed5

File: views/pages/DashboardPage/DashboardPage.test.jsx (hunk 1 above)
Comment (JackVCurtis) [reply views/pages/DashboardPage/DashboardPage.test.jsx]: 3433ed5

File: views/tables/Table/index.jsx (hunk 4 above)
Comment (JackVCurtis) [reply views/tables/Table/index.jsx]: 2052a40

File: test/fixtures.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+import { over } from 'lodash'
```
Comment (mikestone14) [inline test/fixtures.js]: I think this was accidentally imported?

```suggestion
```

File: views/pages/DashboardPage/DashboardPage.test.jsx [hunk 6]
```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { render, screen, within, waitFor } from '@testing-library/react'
+import { AuthContext, NotificationContext } from '../../contexts'
+import { MemoryRouter } from 'react-router-dom'
```
Comment (mikestone14) [inline views/pages/DashboardPage/DashboardPage.test.jsx]: what do you think about organizing these by external/internal import?

```suggestion
import { render, screen, within, waitFor } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import { AuthContext, NotificationContext } from '../../contexts'
```

**Review by mikestone14 — APPROVED** [review]

File: test/fixtures.js (hunk 5 above)
Comment (JackVCurtis) [reply test/fixtures.js]: Aaaagh

Comment (JackVCurtis) [reply test/fixtures.js]: 529a6b0

File: views/pages/DashboardPage/DashboardPage.test.jsx (hunk 6 above)
Comment (JackVCurtis) [reply views/pages/DashboardPage/DashboardPage.test.jsx]: 529a6b0

**Pull request merged**